config/terraform/application/vars/10-video-hearing-wowza/sbox.tfvars

```diff
@@ -27,4 +27,5 @@ route_table = [
     next_hop_type          = "Internet"
     next_hop_in_ip_address = null
   }
-]
\ No newline at end of file
+]
+dynatrace_tenant = "yrk32651"
```

Comment:
We only have Dynatrace for Staging and Production, so other environments need to be empty for the tenant